src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const basicRate = 40;
+
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days >= 3) {
+    if (days >= 7) {
+      return (basicRate * days) - 50;
+    }
```

Comment:
it's better to use 2 separate if statements for readability instead of nested statements